Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml

```diff
@@ -0,0 +1,102 @@
+                    <span class="govuk-label--l">Join company: @organisationDetails.Name</span>
+                </h1>
+                <h2 class="govuk-heading-m">Organisation details</h2>
+                <dl class="govuk-summary-list">
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Organisation name
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">@organisationDetails.Name</p>
+                        </dd>
+                    </div>
+                </dl>
+                <br/>
+                <h2 class="govuk-heading-m">User details</h2>
+                <dl class="govuk-summary-list">
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Firstname
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">John</p>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Lastname
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">Johnson</p>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Email address
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">john@johnson.com</p>
+                        </dd>
+                    </div>
+                </dl>
+            </fieldset>
+        </div>
+    </div>
+    <br/>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form method="post">
+                <fieldset class="govuk-fieldset" data-required="" data-error="Select yes if you want to join this organisation" aria-required="true">
+                    <div class="govuk-form-group @(joinHasError ? "govuk-form-group--error" : "")">
+                        <h2 class="govuk-heading-m">
+                            <span class="govuk-label--m">Would you like to join @organisationDetails.Name?</span>
+                        </h2>
+                        @if (joinHasError)
+                        {
+                            <p id="approval-error" class="govuk-error-message">
+                                <span class="govuk-visually-hidden">Error:</span>
+                                @Html.ValidationMessageFor(m => m.Join)
+                            </p>
+                        }
```

Comment:
The fieldset here needs an aria-describedby attribute that associates it with the error message

Reply:
So `approval-error` ?

Reply:
Done